backend/config.json.example

```diff
@@ -1,14 +1,18 @@
 {
     "target_size": [1220,610],
     "tolerance": 0.01,
-    "company_name": "com.nortd.labs",
+    "company_name": "/dev/ttyACM0",
```

Comment:
Hallo Lukas,

das war wohl ein Problem, als Sven gemerged hat!
Philip hat das Ganmze entdeckt, und company_name durch serial_port ersetzt. wohl aber noch nicht eingecheckt!

Vermutlich stand in der einen Version:

"company_name": "com.nortd.labs"

in der anderen 

"serial_port": "/dev/ttyACM0"

und Sven sah nur den rechten Unterschied.

Wir sollten eine Configdatei machen, die alle möglichen Felder enthält!

Gruß

  Wolfgang


------
In Antwort auf die folgende Mail

From: morbidick <notifications@github.com>
To:   fablab-ka/LasaurApp <LasaurApp@noreply.github.com>
Cc:   
Subject: Re: [fablab-ka/LasaurApp] Smartcard support (#8)
Date:    Thu, 05 May 2016 09:27:14 -0700


> @@ -1,14 +1,18 @@
>  {
>      "target_size": [1220,610],
>      "tolerance": 0.01,
> -    "company_name": "com.nortd.labs",
> +    "company_name": "/dev/ttyACM0",

sure about that?^^

---
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/fablab-ka/LasaurApp/pull/8/files/1a939fea81d939ce9056d923f01c1b1ef82b5749#r62212855